internal/olm/operator/registry/operator_installer.go

```diff
@@ -208,15 +209,17 @@ func (o OperatorInstaller) getOperatorGroup(ctx context.Context) (*v1.OperatorGr
 	return &ogList.Items[0], true, nil
 }
 
-func (o OperatorInstaller) createSubscription(ctx context.Context, cs *v1alpha1.CatalogSource) (*v1alpha1.Subscription, error) {
+func (o OperatorInstaller) createSubscription(ctx context.Context, csName string) (*v1alpha1.Subscription, error) {
 	sub := newSubscription(o.StartingCSV, o.cfg.Namespace,
 		withPackageChannel(o.PackageName, o.Channel, o.StartingCSV),
-		withCatalogSource(cs.GetName(), o.cfg.Namespace),
+		withCatalogSource(csName, o.cfg.Namespace),
 		withInstallPlanApproval(v1alpha1.ApprovalManual))
 
 	if err := o.cfg.Client.Create(ctx, sub); err != nil {
 		return nil, fmt.Errorf("error creating subscription: %w", err)
 	}
+	// TODO(asmacdo) remove dev artifact
+	// color.Yellow("%+v\n", sub)
```

Comment:
Is this comment required? WhY?

Reply:
Its an artifact for development, I left it in just to help me test, but always planned to remove before merge.  Its gone now.